packages/storybook/stories/va-process-list.stories.jsx

```diff
@@ -8,9 +8,14 @@ export default {
   title: 'Components/va-process-list',
   parameters: {
     docs: {
-      /* eslint-disable-next-line react/display-name */
-      page: () => <StoryDocs data={processListDocs} />,
+      description: {
+        component:
+          `<a className="vads-c-action-link--blue" href="https://design.va.gov/components/process-list">View guidance for the Process list component in the Design System</a>` +
+          `\n` +
+          'This component expects `<li>` elements as its children.',
+      },
```

Comment:
Is it possible to update `<StoryDocs>` in a way where we can still rely on the JSDoc comment in the code while also having this action link?

Reply:
We can, but I'm running into issues related to styling that I haven't been able to solve. It seems not all styles are being applied.

```diff
diff --git a/packages/storybook/stories/va-process-list.stories.jsx b/packages/storybook/stories/va-process-list.stories.jsx
index 752b5e0..0bdcdeb 100644
--- a/packages/storybook/stories/va-process-list.stories.jsx
+++ b/packages/storybook/stories/va-process-list.stories.jsx
@@ -8,13 +8,19 @@ export default {
   title: 'Components/va-process-list',
   parameters: {
     docs: {
-      description: {
-        component:
-          `<a className="vads-c-action-link--blue" href="https://design.va.gov/components/process-list">View guidance for the Process list component in the Design System</a>` +
-          `\n` +
-          'This component expects `<li>` elements as its children.',
-      },
+      page: () => (
+        <StoryDocs
+          data={{
+            ...processListDocs,
+            guidance: {
+              componentName: 'Process list',
+              url: 'https://design.va.gov/components/process-list',
+            },
+          }}
+        />
+      ),
     },
+
     componentSubtitle: `Process list web component`,
   },
 };
diff --git a/packages/storybook/stories/wc-helpers.jsx b/packages/storybook/stories/wc-helpers.jsx
index 0302df3..ec6a98b 100644
--- a/packages/storybook/stories/wc-helpers.jsx
+++ b/packages/storybook/stories/wc-helpers.jsx
@@ -100,16 +100,27 @@ export const propStructure = comp => {
   return { ...props, ...events, ...listenersWithDescriptions };
 };
 
+function Guidance({ data: guidance }) {
+  return (
+    <a className="vads-c-action-link--blue" href={guidance.url}>
+      View guidance for the {guidance.componentName} component in the Design
+      System
+    </a>
+  );
+}
+
 /**
  * Return a component with Storybook docs blocks in a standard order.
  * Accepts a JSON object as a prop representing component information
  */
 export function StoryDocs({ data }) {
   const noArgs = data.props.length == 0;
+  const guidance = data.guidance;
   return (
     <>
       <Title />
       <Subtitle />
+      {guidance && <Guidance data={guidance} />}
       <Description markdown={data.docs} />
       <Primary />
       {!noArgs && <ArgsTable story={PRIMARY_STORY} />}

```

This is how it looks:
![Screen Shot 2022-05-04 at 18 06 05](https://user-images.githubusercontent.com/36863582/166833437-7f006634-eda7-4b93-b849-31918b8593dd.png)

This is how it should look (using Modal for example only)
![Screen Shot 2022-05-04 at 18 07 03](https://user-images.githubusercontent.com/36863582/166833527-51a70a5e-c4e6-431b-afeb-ac183ced6e7d.png)

It's not applying these dynamically generated CSS classes:
![Screen Shot 2022-05-04 at 18 07 55](https://user-images.githubusercontent.com/36863582/166833670-a8b33641-9246-4738-96c6-d7c729427405.png)

Reply:
Nevermind, that's the styling that we DO want after consulting Sketch. It's the opposite situation - the current action links underneath the component subtitle are being displayed incorrectly.